tests/.suitespec.json

```diff
@@ -96,6 +96,9 @@
             "ddtrace/internal/ci_visibility/*",
             "ddtrace/ext/test.py"
         ],
+        "llmobs": [
+            "ddtrace/internal/llmobs/*"
```

Comment:
After this change, this path won't exist, right? Maybe you meant `ddtrace/llmobs`?